### ISSUE TITLE
GF broken

### ISSUE THREAD

**Issue by ThaDoctor72** [Pinacolada64/ImageBBS#24]
In non-cursor menu, the commands for editing and adding are broken. Z enters maint menu, but the only command that works in there is O for logoff.
I was able to add and view libraries using the cursor menu however.

**Comment by x-tec2017** [comment]
I'm pretty sure the GF section was originally designed to only allow maintenance functions in cursor menu mode.

**Comment by ThaDoctor72** [comment]
We should then update the menu file to reflect the loss of those commands then.
I would suggest there still be a legacy management feature, in case there are ops like myself who don't use the UI menus.

**Comment by Pinacolada64** [comment]
@ThaDoctor72 , I agree. There should be menus for MN-mode and non-MN-mode, always. Surprisingly, I don't see a line in `im` which checks for graphic menu mode; it could be transplanted from, `IM` which *does* check:

https://github.com/Pinacolada64/ImageBBS/blob/a0b403200187faaaed4e9c9fb125a43db53d7997/v2/core/plus_IM.lbl#L6

but the non-MN mode is not implemented (yet). I want to re-work `IM` so it does implement non-MN mode.

**Comment by ThaDoctor72** [comment]
The menu mode and layout exists, but none of the calls or actual processes.
Almost like it was set up to visually go through the motions, but no
subroutines.
